README.md

```diff
@@ -19,11 +19,11 @@ $ npm install
 ```
 
 ### Config Setup 
```

Comment:
Can add this doc at end of this section https://salesforce.quip.com/bKFnA8hIXOZo. (Can just paste contents or publish doc for external users)